invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -287,9 +294,13 @@ def get_dates(self, obj):
         }]
 
         for date in obj["metadata"].get("dates", []):
+            date_type_id = date.get("type", {}).get("id")
+            props = self._read_date_type(
+                date_type_id
+            ).to_dict().get("props", {})
             to_append = {
                 "date": date["date"],
-                "dateType": date["type"].capitalize()
+                "dateType": props.get("datacite", "Other")
```

Comment:
There is no props defined in the yaml file?

Reply:
Will add them, thanks!